src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -75,7 +75,7 @@
                        (assoc token
                               :networks           (utils/network-list token networks)
                               :total-balance      (utils/total-token-value-in-all-chains token)
-                              :total-balance-fiat (utils/calculate-balance token)))
+                              :total-balance-fiat (utils/calculate-balance-for-token token)))
```

Comment:
Great it's solved, but since the PR involving bignumber balances has  been merged, I'd suggest using an implementation based on bignumbers :+1:

Reply:
thanks, updated!